cmd/txtar-addmod/script_test.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/rogpeppe/go-internal/goproxytest"
+	"github.com/rogpeppe/go-internal/gotooltest"
+	"github.com/rogpeppe/go-internal/testscript"
+)
+
+var proxyURL string
+
+func TestMain(m *testing.M) {
+	os.Exit(testscript.RunMain(gobinMain{m}, map[string]func() int{
+		"txtar-addmod": mainint,
+	}))
+}
+
+type gobinMain struct {
+	m *testing.M
+}
+
+func (m gobinMain) Run() int {
+	if os.Getenv("GO_GCFLAGS") != "" {
```

Comment:
I think we don't need lines 26 to 32 here.

Reply:
Done